theme/css/general.css

```diff
@@ -0,0 +1,279 @@
+/* Base styles and content styles */
```

Comment:
It's strange that we've been missing it all along :smile: 

My understanding is that `mdbook build` will pull in theme files from https://github.com/rust-lang/mdBook/tree/master/src/theme/css, unless the local theme has a file with the same name.

But perhaps I've misunderstood this all along?